src/js/components/LeftPane.js

```diff
@@ -20,6 +25,18 @@ const Arrow = (props) => {
   )
 }
 
+const StyledSection = styled.section`
+  overflow: hidden;
+  position: relative;
+  min-height: 24px;
+`
+
+const SectionContents = styled.div`
+  height: 100%;
+  display: ${(props) => (props.show ? "block" : "none")};
+  overflow-y: scroll;
+`
+
 const SectionHeader = styled.div`
   display: flex;
```

Comment:
If you would add a `user-select: none` here, that would make it feel more native as well. That when when a user is interacting with it, it won't make a selection:

<img width="289" alt="image" src="https://user-images.githubusercontent.com/3460638/85463959-fb3dbc80-b55b-11ea-9fe5-2076f23d89c8.png">